components/AppManager.jsx

```diff
@@ -4,32 +4,28 @@ import {Routes, Route} from 'react-router-dom'
 
 import ArticlesList from './ArticlesFeed/ArticlesList'
 
-import NavBar from "./navBar/NavBar";
-
-import ChosenArticlePage from "./chosenArticlePage/ChosenArticlePage";
+import ChosenArticlePageManager from "./chosenArticlePage/ChosenArticlePageManager";
 
+import fetchAllArticles from "../utils/fetchAllArticles";
 
 export default function AppManager () {
     const [articles, setArticles] = useState()
     const [chosenArticleId, setChosenArticleId] = useState("")
-console.log(chosenArticleId)
+
+
+
     useEffect(() => {
-        const fetchAllArticles = async () => {
-            const articleQuery = await fetch(`https://be-nc-news-famm.onrender.com/api/articles`) 
-            const response = await articleQuery.json()
-            setArticles(response.articles)
-        }
-        fetchAllArticles()
-    },[setChosenArticleId])
+        fetchAllArticles(setArticles)
+    },[])
 
     return (
         <>
-        <NavBar/>
+        {/* <NavBar/> */}
         <Routes>
             <Route path="/" element={
                 <ArticlesList articles={articles} setChosenArticleId={setChosenArticleId}/>
             }/> 
-        <Route path={`/articles/${chosenArticleId.article_id}`} element={<ChosenArticlePage chosenArticleId={chosenArticleId}/>}/>
+        <Route path={`/articles/${chosenArticleId.article_id}`} element={<ChosenArticlePageManager chosenArticleId={chosenArticleId}/>}/>
```

Comment:
I'm afraid this route isn't working you need to use a parametric endpoint
https://notes.northcoders.com/courses/js-front-end/react-router#parametric-endpoints-useparams 
When I navigate to /articles/34 it says route not found, the only way to get to your article is via clicking on an article in your ArticlesList, when you refresh the page it breaks.
This isn't connected to this ticket but certainly worth fixing.